### DISCUSSION TITLE
How often are the GitHub Runner Images updated?

### DISCUSSION THREAD

**Discussion by TimWolla** [ubicloud/ubicloud#1826] [Q&A] [answered]
Hi

I am seeing that my `ubicloud-standard-2-ubuntu-2204` runner is using the 20240630 version of the ubuntu-22.04 image:

```
Version: 20240630.1.1
Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20240630.1/images/ubuntu/Ubuntu2204-Readme.md
```

This version comes with Docker Buildx 0.15.1, which unfortunately does not yet include the fix for https://github.com/docker/buildx/pull/2558.

Docker Buildx 0.16 was first included in version 20240714 (https://github.com/actions/runner-images/commit/c2c7c501c828d25ead9fa708c055352ea47201c5) and the current version is 20240730.

How often are the runner images generally updated for Ubicloud and more specifically, when can I next expect an update to an image that includes the newer Buildx version?

**Comment by enescakir** [comment, marked as answer]
Hi @TimWolla, thank you for your question. We typically update Ubicloud images a few weeks after GitHub deploys a new version. In the past, we've seen instances where GitHub released problematic versions and had to quickly roll them back. Our aim is to maintain a balance.
We plan to deploy the new version `20240721.1.0` this week.
